index.js

```diff
@@ -0,0 +1,232 @@
+//Problem 1
+
+// document.addEventListener("DOMContentLoaded",() =>{
+//     let name = prompt("What is your name? ")
+//     document.querySelector("#yourName").innerText = `${name}`
```

Comment:
You don't need string interpolation here, you can just set the variable to the innerText